Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -722,6 +722,97 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Resolve Conflict SG", "[.SyncServe
     }
 }
 
+TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Auto Purge Enabled - Filter Revoked Revision - SGColl", "[.SyncServerCollection]") {
+    _authHeader = "Basic c2d1c2VyOnBhc3N3b3Jk"_sl; // Admin user for REST requests
+    const string docIDstr = timePrefix() + "apefrr-doc1";
+
+
+    HTTPStatus status;
+    C4Error error;
+    sendRemoteRequest("POST", "_user", &status, &error, R"({"name":"filterRevoked","password":"password"})", true);
+    sendRemoteRequest("PUT", "_user/filterRevoked", &status, &error, R"({"admin_channels":["*"]})", true);
```

Comment:
(Not for this PR) Maybe we should create a SG class that does all of the REST API with the APIs such as createUser(), deleteUser(), addDoc(s), getDoc(s), getDocIDs(), getDocCount(), purgeDoc() and etc.

Reply:
We absolutely should, and I think this is a good collaboration for @velicuvlad and @callumbirks to work on.  If there is a certain action that starts to take multiple steps each time it can be brought into the macro server as well.